scipy/stats/stats.py

```diff
@@ -601,7 +623,7 @@ def hmean(a, axis=0, dtype=None):
         raise ValueError("Harmonic mean only defined if all elements greater than zero")
 
 
-def mode(a, axis=0):
+def mode(a, axis=0, nan_policy='propagate'):
```

Comment:
`mode`'s `nan_policy` default should probably be 'omit' for backwards compatibility.